src/FSharpx.Extras/ComputationExpressions/Monad.fs

```diff
@@ -1122,6 +1122,14 @@ module Task =
     /// Creates a task that runs the given task and ignores its result.
     let inline Ignore t = bind (fun _ -> returnM ()) t
 
+    /// Active pattern that matches on flattened inner exceptions in an AggregateException
+    let (|AggregateExn|_|) (e:exn) =
+        match e with
+        | :? AggregateException as ae -> ae.Flatten().InnerExceptions
+                                         |> List.ofSeq
+                                         |> Some
+        | _                           -> None
```

Comment:
This is just a style preference, but I'm not a big fan of having the code in pattern matching aligned after `->`. I think it creates unnecessary indentation. So, I'd prefer something like:

    | :? AggregateException as ae -> 
        ae.Flatten().InnerExceptions |> List.ofSeq |> Some
    | _  -> None

Just a style thing though, so feel free to ignore this :-)